src/FilterBundle/Helper/EnumAttributeHelper.php

```diff
@@ -47,41 +70,51 @@ public function __construct(Client $client)
      */
     public function getLabels(string $attributeName): array
     {
-        $productTypeRequest = ProductTypeQueryRequest::of();
+        $cacheItem = $this->cacheItemPool->getItem(self::CACHE_KEY . '-' . md5($attributeName));
 
-        $response = $this->client->execute($productTypeRequest);
+        if ($cacheItem->isHit()) {
+            $labels = $cacheItem->get();
+        } else {
+            $productTypeRequest = ProductTypeQueryRequest::of();
```

Comment:
on cache warmup you are firing this endpoint every often. internal property storage after first run would help. you just need one call per request scope to this api.